libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route/ReplayRouteOptions.kt

```diff
@@ -80,7 +80,7 @@ class ReplayRouteOptions private constructor(
         private var maxSpeedMps = 30.0
         private var turnSpeedMps = 3.0
         private var uTurnSpeedMps = 1.0
-        private var maxAcceleration = 4.0
+        private var maxAcceleration = 3.0
```

Comment:
4 m/s^2 is equivalent to going 0-60 mph in **6.6** seconds.

3 m/s^2 is equivalent to going 0-60 mph in **8.94** seconds.

Slowing down the speed of replay to lower the changes of map matching to nearby roads
https://github.com/mapbox/bmw-collab/issues/432#issuecomment-691917006